### PR TITLE
created email notification file

### PR DESCRIPTION
**Pull request by michael-staples**
created server folder and email notification file within it.

### CODE REVIEW

**Comment by simpsbri** [comment]
lgtm

**Review by simpsbri — APPROVED** [review]
lgtm

**Comment by michael-staples** [comment]
Added email notification and input validation for project submissions. Feedback on emailNotification.js error handling appreciated!

File: capstone-team-1/server/emaillNotification.js [hunk 1]
```diff
@@ -0,0 +1 @@
+g;
```
Comment (dandigangi) [inline capstone-team-1/server/emaillNotification.js:1]: wut

Comment (simpsbri) [reply capstone-team-1/server/emaillNotification.js:1]: We had some merge issues and code commit issues that we were attempting to push files that had no changes in them and we added a single line to simply trigger the need to update the file.

**Pull request merged**